Automattic/wp-calypso
# Posts: Dereduxify post type list likes

This is a follow-up to https://github.com/Automattic/wp-calypso/pull/45210#discussion_r478233050 cc @jsnajdr.

#### Changes proposed in this Pull Request

* Posts: Remove redux implementation for post action count likes in favor of component state
* State: Remove unnecessary post action count likes state (actions, reducer, selectors)
* State: Convert post type list tests to use jest instead of chai

#### Testing instructions

* Go to http://calypso.localhost:3000/posts/:site where `:site` is a public site with some traffic.
* Play with the popover that appears when you click on "% Likes" of a post, and verify it works like it did before:
  * Clicking it initially opens the popover.
  * Clicking anywhere closes the popover.
  * Clicking on another popover closes the previous popover in favor of the new one.
* Verify tests still pass.
